airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.java

```diff
@@ -227,12 +230,53 @@ void testCheckSuccess(final DatasetIdResetter resetDatasetId) {
   void testCheckFailure(final DatasetIdResetter resetDatasetId) {
     ((ObjectNode) config).put(BigQueryConsts.CONFIG_PROJECT_ID, "fake");
     resetDatasetId.accept(config);
-    final AirbyteConnectionStatus actual = new BigQueryDestination().check(config);
-    final String actualMessage = actual.getMessage();
+
+    // Assert that check throws exception. Later it will be handled by IntegrationRunner
+    final Exception ex = assertThrows(RuntimeException.class, () -> {
```

Comment:
can you specifically test for ConfigErrorException here?

Reply:
Updated, thanks